lib/config.js

```diff
@@ -1,18 +1,25 @@
 'use strict';
 
+/**
+ * Configuration module.
+ * @module config
+ */
+
 // Modules
 var _ = require('lodash');
 var fs = require('fs-extra');
 var os = require('os');
 var path = require('path');
 var yaml = require('js-yaml');
 
-/*
- * Document
+/**
```

Comment:
@jmdrawneek ditto re `/**` here. I think we only want to use the double star comment blocks for exported functions